lib/activerecord-import/base.rb

```diff
@@ -16,6 +16,7 @@ def self.base_adapter(adapter)
     when 'makara_postgis' then 'postgresql'
     when 'postgis' then 'postgresql'
     when 'cockroachdb' then 'postgresql'
+    when 'trilogy' then 'trilogy'
```

Comment:
You don't need this line. The next line does that, given your other files added.

Reply:
You're right. I'll remove it.